packages/core/src/index.tsx

```diff
@@ -11,23 +11,14 @@ import { ProxyConfiguration, ProxyType } from './ProxyConfiguration';
 import { SdkVerbosity } from './SdkVerbosity';
 import { TrackingConsent } from './TrackingConsent';
 import { DdLogs, DdTrace, DdRum } from './foundation';
-import type { RumActionType, ResourceKind, ErrorSource } from './types';
-
-/**
- * Using a syntax like `RumActionType as DdRumActionType` breaks unit tests
- * as babel does not seem to be able to recognize the type alias.
- */
-type DdRumActionType = RumActionType;
-type DdResourceKind = ResourceKind;
```

Comment:
what about this one? doesn't seem to be exported

Reply:
Actually this one is a union type, not an enum. So you don't import it when specifying a value that has this type.